packages/tc-schema-sdk/lib/updater.js

```diff
@@ -27,7 +44,10 @@ class SchemaUpdater {
 		if (schemaDirectory && !schemaData) {
 			schemaData = {
 				schema: {
-					types: readYaml.directory(schemaDirectory, 'types'),
+					types: [].concat(
+						readYaml.directory(schemaDirectory, 'types'),
+						readRelationships(schemaDirectory),
```

Comment:
can relationship types go in a separate top level property of `relationshipTypes`. They are rarely (never?) useful to have in a combined list, so woudl prefer to always keep them separate.

Reply:
Then can we define root relationship types in top level like:

```
schema: {
  types: ...,
  relationships: ...,
  ...,
}
```

I thought that the schema structure shouldn't change for backward compatibility.
But if I can do that, data accessors can be more simpler.